### DISCUSSION TITLE
2020/02/avoid-these-blue-green-deployment-pitfalls/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#743] [Specific Post Comments]
# 2020/02/avoid-these-blue-green-deployment-pitfalls/

I’ve seen organizations make some …interesting decisions around the concept of “blue/green deployments.” In some cases, these changes can lead to some pretty big pitfalls, so I wanted to write up some thoughts here to help folks avoid them.

https://seankilleen.com/2020/02/avoid-these-blue-green-deployment-pitfalls/

**Comment by jfneis** [comment]
Thank you! Event with 2+ years old, this posted helped us a lot assuring we are not missing any hidden threats in our adoption of Blue Green Deployment!

**Reply by SeanKilleen** [reply]
Thanks for leaving a comment! Glad to hear that it helped. :)
